condor_watch_q.py

```diff
@@ -466,12 +523,24 @@ def watch_q(
 
     try:
         msg, move, clear = None, None, None
+        batch_name_to_nodes_total = {}
```

Comment:
I'd like to get the initial value of this from the initial query, so that we can still do only a single query if all of the information is already available in the first one.